### DISCUSSION TITLE
how to get upstream host and port to log?

### DISCUSSION THREAD

**Discussion by jianmaikj** [Kong/kong#9388] [Help] [answered]
upstream_uri in kong.log.serialize() does not contain these information

**Comment by shettyh** [comment, marked as answer]
`kong.log.serialize()` already has these data here.
https://github.com/Kong/kong/blob/ef153cf33b239923ee02a54f2fe3f6b829151bac/kong/pdk/log.lua#L712

This will be a array, and each item will have `ip` and `port` information

**Reply by jianmaikj** [reply]
ok,thank you!
